### DISCUSSION TITLE
Como adicionar imagem no readme do GitHub

### DISCUSSION THREAD

**Discussion by MarceloSantana25** [community/community#66495] [Programming Help] [answered]
### Select Topic Area

Show & Tell

### Body

# Essa é a melhor maneira de colocar imagens? Qual  número da cor vermelha para deixar o logo igual da marca?
# Adicionar imagens no readme do GitHub:
[![Youtube](https://img.shields.io/badge/yuotube-000?style=for-the-badge&logo=youtube&logoColor)](https://www.youtube.com/watch?v=oMFiGEZ6UlQ)

**Comment by LeandroLuna** [comment, marked as answer]
Olá, Marcelo!

Você pode adicionar imagens de duas maneiras comuns:

1. **Incorporar a imagem diretamente no README:**

   Você pode incorporar uma imagem diretamente no README usando a seguinte sintaxe de Markdown:

   ```markdown
   ![Texto Alternativo](URL_da_Imagem)
   ```

   - `Texto Alternativo`: Isso é o que será exibido se a imagem não puder ser carregada ou para fins de acessibilidade. É uma boa prática descrever brevemente a imagem aqui.
   - `URL_da_Imagem`: Substitua isso pela URL da imagem que deseja incorporar. A imagem deve estar hospedada online, como no próprio repositório, em um serviço de hospedagem de imagens, ou em qualquer outro local acessível pela internet.

   Por exemplo:

   ```markdown
   ![Logo da Minha Empresa](https://exemplo.com/logo.png)
   ```

2. **Usar a tag HTML `<img>`:**

   Você também pode usar a tag HTML `<img>` para inserir uma imagem diretamente no README. Isso é útil quando você precisa de mais controle sobre o tamanho ou outros atributos da imagem. O exemplo a seguir demonstra como fazer isso:

   ```html
   <img src="URL_da_Imagem" alt="Texto Alternativo">
   ```

   - `src`: Substitua isso pela URL da imagem.
   - `alt`: Como na sintaxe Markdown, isso é o texto alternativo que será exibido se a imagem não puder ser carregada ou para fins de acessibilidade.

   Por exemplo:

   ```html
   <img src="https://exemplo.com/logo.png" alt="Logo da Minha Empresa">
   ```

Dito isso, a primeira opção (usando Markdown) é mais comum para READMEs simples, enquanto a segunda (usando HTML) oferece maior flexibilidade de formatação.

**Reply by LeandroLuna** [reply]
Em relação a segunda pergunta, 'Qual número da cor vermelha para deixar o logo igual da marca?', não compreendi direito. Poderia especificar melhor, por favor?
